apps/src/templates/sectionProgress/progressTables/ProgressTableLevelBubble.jsx

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import Radium from 'radium';
+import PropTypes from 'prop-types';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import i18n from '@cdo/locale';
+import * as progressStyles from '@cdo/apps/templates/progress/progressStyles';
+import color from '@cdo/apps/util/color';
+import {LevelStatus} from '@cdo/apps/util/sharedConstants';
+
+const styles = {
+  container: {
+    ...progressStyles.flex,
+    width: progressStyles.BUBBLE_CONTAINER_WIDTH
+  },
+  main: {
+    ...progressStyles.flex,
+    ...progressStyles.font,
+    boxSizing: 'content-box',
+    width: progressStyles.DOT_SIZE,
```

Comment:
Would it make sense to split out a `bigCircle` style?

Reply:
yes i agree that will be clearer. i'd also like to rename some of those size values but i'll save that for when we refactor `ProgressBubble`